### DISCUSSION TITLE
Where do i publish plugins?

### DISCUSSION THREAD

**Discussion by NezbednikSK** [Pryaxis/TShock#1723] [General]
Since the forums are retired now, where do i publish plugins?
I am working on one, and i can't figure out where to post it.

**Comment by moisterrific** [comment]
Just publish it as a release on GitHub and tag it with TShock plugin or something. If the forum rules still stand then it should be open source where everyone can inspect or fork the project.

**Comment by hakusaro** [comment]
For now, the best thing is to publish on GitHub and tag it with `tshock-plugin`. We're working on a replacement for this. If you have a plugin submission you want released, send me an email (shank@shanked.me) with your forum user name and I can help you get a resource posted if you need to before we have a backup solution. People can browse plugins via the topics, like https://github.com/topics/tshock-plugin.

**Comment by NezbednikSK** [comment]
Thanks, i will surely do that!

**Comment by hakusaro** [comment]
Please submit a plugin here: https://github.com/Pryaxis/Plugins. We're going to be using this as the canonical source of truth for plugins going forward until we have a better solution in place.
